conda.recipe/conda_build_config.yaml

```diff
@@ -33,3 +33,25 @@ zip_keys:                       # [osx]
   -                             # [osx]
     - macos_min_version         # [osx]
     - MACOSX_DEPLOYMENT_TARGET  # [osx]
+
+
+# pin linux compiler to old build before the sysroot change because clang 9 wasn't updated
+# to work with latest build of gcc
+c_compiler:
+  - gcc                        # [linux]
+  - clang                      # [osx]
+  - vs2017                     # [win]
+c_compiler_version:            # [unix]
+  - 11                         # [osx and arm64]
+  - 10                         # [osx and x86_64]
+  - 7.5.0 h09487f9_20          # [linux64 or aarch64]
```

Comment:
If you want to lock down the gcc compiler to a specific version, you need to do this.  I'm also now realizing why conda is having trouble solving our environment on Mac.  The recipe asks for clang 10 and clang 9.  I'm going to let the CI run once to make sure before I push more commits to fix that.

Reply:
Looks like it's indeed asking for both.